src/ports/orders/utils.ts

```diff
@@ -219,10 +219,6 @@ export function fromOrderFragment(
 
   let expiresAt = fragment.expiresAt
 
-  if (expiresAt.length < 13) {
```

Comment:
This will return in seconds now, has to be handled by the UI to prevent inconsistencies. I would prefer adding a separate field to the order like orderType: 1,2 and differentiate it in the UI with that to prevent breaking the api.

Not  a blocker